contracts/token/ERC20/MintableToken.sol

```diff
@@ -41,10 +41,8 @@ contract MintableToken is StandardToken, Ownable {
     canMint
     returns (bool)
   {
-    totalSupply_ = totalSupply_.add(_amount);
-    balances[_to] = balances[_to].add(_amount);
     emit Mint(_to, _amount);
-    emit Transfer(address(0), _to, _amount);
+    _mint(_to, _amount);
```

Comment:
Shouldn't the event be emitted after the action took place, according to our guideline?

Reply:
Ah, yes. I initially did it like this because the test was relying on the previous order: `Mint` followed by `Transfer`. I'll fix the test.